src/utils/cache.ts

```diff
@@ -107,3 +107,42 @@ export async function getBlockedAddresses(redis: Redis, URL: string): Promise<st
 
   return blockedAddresses
 }
+
+export async function getRateLimitedApps(redis: Redis, URL: string): Promise<string[]> {
+  if (URL === '') {
+    return []
+  }
+  const rateLimitedAppsKey = 'rateLimitedApps'
+  const cachedRateLimitedApps = await redis.get(rateLimitedAppsKey)
+  let rateLimitedApps: string[] = []
+
+  if (!cachedRateLimitedApps) {
+    try {
+      const axiosConfig = {
+        method: 'GET',
+        url: URL,
+        timeout: 10000,
+      } as AxiosRequestConfig
+
+      const { data } = await axios(axiosConfig)
+      const { ApplicationIDs: rateLimitedAppsList } = data
```

Comment:
```suggestion
      const { applicationIDs: rateLimitedAppsList } = data
```

Reply:
This has to stay upper-case to match the output of `rate-limiter`(otherwise we will miss the output)

Reply:
All right